docs/reference/setup/install/docker.asciidoc

```diff
@@ -143,9 +167,9 @@ ifeval::["{release-state}"!="unreleased"]
 --------------------------------------------
 version: '2'
 services:
-  elasticsearch1:
-    image: docker.elastic.co/elasticsearch/elasticsearch:{version}
-    container_name: elasticsearch1
+  elasticsearch:
+    image: {docker-image}
```

Comment:
Renaming this container to the default name makes it easier to add other Stack components without having to explicitly configure them to hit `elasticsearch1`. This also applies to Kibana and Logstash monitoring, which work out-of-the-box with this change.